pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml

```diff
@@ -4,10 +4,11 @@ model:
   feature_dim: 512
   fpn: True
   fpn_dim: 256
+  fpn_process: 'max_pooling'
 
 data:
   type: 'image'
-  sources: ['msmt17', 'market1501']
```

Comment:
Should config for 200 model be changed?
And how about update readme and add checkpoint for new model?

Reply:
`sources` in 0200 model config reverted. Readme file updated with the new 0300 model.

Reply:
May be we should delete 0200 model? New 0300 model replaces 0200 one that also is removed from upcoming OpenVINO release.

Reply:
Yes, if 300 model replaced 200 that you can remove.

Reply:
0200 model removed.